symengine/lib/symengine_wrapper.pyx

```diff
@@ -785,6 +787,31 @@ cdef class Basic(object):
     def is_Matrix(self):
         return False
 
+    def copy(self):
+        return self.func(*self.args)
+
+    def xreplace(self, rule):
+        value, _ = self._xreplace(rule)
+        return value
+
+    def _xreplace(self, rule):
```

Comment:
This method is slow. Use `subs` to do this and check that the result is different.